packages/components/input-number/style/themes/default.variable.less

```
@@ -1,3 +1,5 @@
 @input-number-color-disabled: @input-color-disabled;
 @input-number-icon-color-hover: @input-icon-color-hover;
 @input-number-invalid-color: @form-invalid-color;
+@input-number-invalid-color: var(--ix-color-red);
+@input-number-invalid-box-shadow: 0 0 0 2px fade(@color-red, 20%);
```

```suggestion
@input-number-invalid-box-shadow: @form-invalid-box-shadow;
```
invalid-color 已经有变量了，不需要重复定义。
另外这个改动我需要和设计师沟通一下。